data/human/fleets.txt

```diff
@@ -2522,6 +2524,13 @@ fleet "pirate raid"
 		"Berserker (Afterburner)" 3
 	variant 2
 		"Berserker (Afterburner)" 2
+	variant 2
+		"Berserker (Twin Modified Blaster)" 2
+	variant 8
+		"Berserker (Twin Modified Blaster)" 3
+	variant 8
+		"Berserker (Twin Modified Blaster)" 2
+		"Berserker (Afterburner)" 2
```

Comment:
I feel like these should all be changed; far too common. 3 Berserkers with Twin Modified Blasters are more common than 2 Berserkers with Twin Modified Blasters. I know you probably took it from the Afterburner Berserkers, but change that too.

Reply:
Yeah, i made it pretty much the same as the Afterburner variant so it appeared about as often. I did notice it was odd 3 Berserkers were more common than 2 in that fleet, but didn't think to change it - I'll flip weights around.

For the reverse-thrust variant, i guess it just being kind of... bad, does serves to show the model's age, yeah, so i'll implement it in smaller numbers to the others. Thanks for the suggestion! 

Also, i actually remember your other variants, they are very nice, but as this PR was more to fix and improve the current variants than add new ones, i didn't think to ask if i could borrow those. Now that we're doing the latter as well, i could add them, if you'd be ok with that! Or we can leave them for you to deal with them later, as you said. They're yours, so that's up to you, but i will say i like them.

Reply:
There, added the rever thrust variant and changed the weights.